stylus/ui-components/alerts.styl

```diff
@@ -30,6 +29,7 @@ $alerts
 
     .coz-alert
         display          block
+        z-index          100
```

Comment:
Honestly I don't know, I'm not the one who wrote it. Maybe a cleaning would be a good thing but not right now. And I have some kind of a Z-Index manager in my todo list, so we'll see that later.
So, if it's a bugfix then I'm good :)